<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustWindow</class>
 <widget class="QMainWindow" name="CustWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenAcc">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 248, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Открыть счетъ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseAcc">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 248, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Закрыть счетъ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddCredit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>590</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 248, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Оформить ​заемъ​</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
background-color: rgba(255, 119, 121, 255);</string>
    </property>
    <property name="text">
     <string>Выходъ</string>
    </property>
   </widget>
   <widget class="QLabel" name="Acc_lable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Счета:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Credit_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Заемы:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLine">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="SurnameLine">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>771</width>
      <height>641</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>-10</y>
      <width>1071</width>
      <height>1011</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Images.qrc">:/LoginImage/resource/images/Sena-Saber-5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Request">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 248, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Заявка</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="AccTable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>421</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BalanceEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Balance_lable">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Балансъ: </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DelSpin">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>280</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="CreditTable">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>421</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CashOut">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 248, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Снять деньги со счета</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSum">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 248, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Пополнить счет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Transfer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 248, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Перевести средства</string>
    </property>
   </widget>
   <widget class="QLabel" name="Balance_lable_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Фамилія: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Balance_lable_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>280</y>
      <width>1071</width>
      <height>1011</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Images.qrc">:/LoginImage/resource/images/Sena-Saber-5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-190</x>
      <y>20</y>
      <width>471</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Images.qrc">:/LoginImage/resource/images/pngwing.com.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>Background</zorder>
   <zorder>OpenAcc</zorder>
   <zorder>CloseAcc</zorder>
   <zorder>AddCredit</zorder>
   <zorder>Exit</zorder>
   <zorder>Acc_lable</zorder>
   <zorder>Credit_label</zorder>
   <zorder>NameLine</zorder>
   <zorder>SurnameLine</zorder>
   <zorder>Request</zorder>
   <zorder>AccTable</zorder>
   <zorder>BalanceEdit</zorder>
   <zorder>Balance_lable</zorder>
   <zorder>DelSpin</zorder>
   <zorder>CreditTable</zorder>
   <zorder>CashOut</zorder>
   <zorder>AddSum</zorder>
   <zorder>Transfer</zorder>
   <zorder>Balance_lable_2</zorder>
   <zorder>Balance_lable_3</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
